libs/platform/src/lib/form/multi-combobox/multi-combobox-mobile/multi-combobox/multi-combobox-mobile.component.html

```diff
@@ -1,16 +1,17 @@
 <ng-template let-dialog let-dialogConfig="dialogConfig" #dialogTemplate>
     <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
-        <fd-dialog-header>
-            <h1 fd-title *ngIf="mobileConfig?.title" [innerText]="mobileConfig.title"></h1>
+        <fd-dialog-header aria-labelledby="fd-title">
+            <h1 id="fd-title" fd-title *ngIf="mobileConfig?.title" [innerText]="mobileConfig.title"></h1>
```

Comment:
You can't hardcode the id values. It introduces 2 problems:
1. If the user wants to set a custom id
2. i18n

Reply:
Will fix.